tests/testlib/s2n_testlib.h

```diff
@@ -166,6 +166,22 @@ S2N_RESULT s2n_connection_set_test_master_secret(struct s2n_connection *conn, co
 
 #define S2N_DEFAULT_TEST_DHPARAMS S2N_DHPARAMS_2048
 
+/* CRL testing files */
```

Comment:
If these aren't needed by other tests, why put them in testlib?

Reply:
I moved these to s2n_crl_test